.github/workflows/release.yml

```diff
@@ -50,6 +50,16 @@ jobs:
         sudo rm -rf "/usr/local/share/boost"
         sudo rm -rf "$AGENT_TOOLSDIRECTORY"
 
+    - name: Install Microsoft Go
+      run: |
+        GO_VERSION=$(sed -n 3p ./go.mod | awk '{ print $2; }')
```

Comment:
Just a nitpick, but here's a safer alternative if you can run the "Install Go" job first:
```
go mod edit -json | jq -r .Go
```

Reply:
Awesome, updated to use that